meson.build

```diff
@@ -93,6 +93,32 @@ conf_data = configuration_data()
 conf_data.set10('CAGE_HAS_XWAYLAND', have_xwayland)
 conf_data.set_quoted('CAGE_VERSION', version)
 
+scdoc = dependency('scdoc', version: '>=1.9.2', native: true)
```

Comment:
Can you make it configurable like https://github.com/swaywm/swayidle/commit/0cdb502d208a? Downstream may want Cage build to be deterministic.

Reply:
Sure but the main author says no https://github.com/Hjdskes/cage/issues/137#issuecomment-611507781

Reply:
@jbeich How do man pages make the build nondeterministic?

Reply:
Installed files. A separate option makes it possible to quickly abort build if scdoc is not found or not usable (e.g., broken binary) instead of installing scdoc and hope meson picks it up.

Reply:
Downstream may also forget to add scdoc dependncy or accidentally remove it via typo/mismerge/etc.

Reply:
Makes sense, I suppose. @travankor can you make manpage generation/installation optional, defaulting to true?